### DISCUSSION TITLE
tracking link was denied

### DISCUSSION THREAD

**Discussion by oussamaelwarraki** [postalserver/postal#2438] [Help with using Postal]
Access to click.luzinmedia.com was denied
You don't have authorization to view this page.
HTTP ERROR 403

this is my Caddyfile : 

`mail.luzinmedia.com {
  reverse_proxy 127.0.0.1:5000
}

click.luzinmedia.com {
  tls it@luovatechnologie.com
  reverse_proxy 127.0.0.1:5000 {
    header_up X-Postal-Track-Host "1"
  }
}
`

**Comment by uhlhosting** [comment]
@oussamaelwarraki you are not helping to get your issue solve by adding a ISSUE and a discussion, you just cause noise unneeded. Please remove the issue, and wait here until someone will reply.

**Comment by uhlhosting** [comment]
@oussamaelwarraki do you have proper DNS settings place? It seems a DNS issue at first sight.

**Comment by oussamaelwarraki** [comment]
Should i create a Cname for click.luzinmedia.com to point at track.mail.luzinmedia.com
Or point mail.luzinmedia.com dorectly to my ip via A record

**Comment by willpower232** [comment]
duplicate #2415

**Comment by oussamaelwarraki** [comment]
@willpower232  not the same this is my caddy file and dns records


Dns : 

A | click | 153.92.209.3 | 14400
A | mail | 153.92.209.3 | 14400


Caddyfile:

`

mail.luzinmedia.com {
        reverse_proxy 127.0.0.1:5000
}


click.luzinmedia.com {
        tls admin@luzinmedia.com
        reverse_proxy 127.0.0.1:5000 {
                header_up X-Postal-Track-Host "1"
        }
}
`

**Comment by oussamaelwarraki** [comment]
and what's this in the postal interface 
"his is the domain that requests for tracked links will be directed through when you use click tracking. We recommend using something like click.yourdomain.com. You will need to a CNAME record to point to track.mail.luzinmedia.com for this once you've added it."
